src/components/chatlog/index.tsx

```diff
@@ -6,21 +6,22 @@ import { PublishEntry } from './PublishEntry';
 type Props = {
   log: Entry[];
   setCurrentMessage: (message: string) => void;
+  setDebugEntry: (entry: Entry) => void;
   handlePublish: (message: string) => Promise<void>;
 };
 
 export const Chatlog = ({
   log,
   setCurrentMessage,
+  setDebugEntry,
   handlePublish,
 }: Props): JSX.Element => (
   <div className="panel-two">
     <h2>Entry Log</h2>{' '}
     <div className="messages">
       {log.reverse().map((entry) => (
-        <div key={entry.entry_hash}>
-          <h3>Hash {entry.entry_hash.slice(0, 8)}</h3>
-          <SyntaxHighlighter>{entry.debugDecoded}</SyntaxHighlighter>
+        <div key={entry.entry_hash} onClick={() => setDebugEntry(entry)}>
+          <h3>Entry {entry.entry_hash.slice(0, 8)}</h3>
```

Comment:
Maybe we use the last 8 bytes instead of the first since they are not really significant (look all similar due to the 2 initial YAMF bytes)?

Reply:
ahhh that's why!

Reply:
I totally forgot about those